pandas/core/algorithms.py

```diff
@@ -724,7 +724,7 @@ def value_counts(
         result = result.sort_values(ascending=ascending)
 
     if normalize:
-        result = result / float(counts.sum())
+        result = result / float(sum(counts))
```

Comment:
This is using the python sum function? This operates on an iterable and this is not vectorised. How will this impact performance?

Reply:
This will be slow, so I change Python sum function to ndarry sum function.